README.md

```diff
@@ -1,62 +1,21 @@
-# NEAR Protocol Stateless Validation available for public preview
+# NEAR Stake Wars IV: Attack of Transactions
 
-NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
+NEAR Protocol is preparing a major update of the core protocol architecture that takes scalability to the next level. This update is called stateless validation, and it has ample impact on how the network operates internally. We invite everyone to join an early access network that runs on this new architecture, StatelessNet, and run a stateless validator node or test the network throughput and scalability.
 
-## What are the upcoming changes?
-In Q2 2024, NEAR protocol will introduce the biggest upgrade since its mainnet launch in how the protocol operates, Stateless validation. The upgrade includes the following features:
-* Stateless validation: Validators do not have to track all shards anymore and can retrieve everything they need to validate state change (state witness) over the network. With stateless validation, NEAR can finally achieve true form of sharding and hardware requirements for a validator node can be dramatically reduced. State validation does not require local tracking of any shards as nodes can retrieve everything they need to validate state change over the network. As a result, hardware requirements for a validator node can be dramatically reduced.
-* In memory trie: Validator nodes can load the entire state into RAM for maximized performance. With minimized storage access, transaction throughput will be greatly improved. 
+You can find the stats [here](https://nearinc.grafana.net/public-dashboards/ebf4227a92de4bb5b3e02938129be6ae).
```

Comment:
As per our discussion, let's include the dashboard next week after the network is up and running and is stabilised.